public/member_ops.js

```diff
@@ -154,6 +154,29 @@ function handle_members(event) {
       });
     } else if (event.target === deleteMember) {
     // Your code goes here!
+    let headers = { "Content-Type": "application/json" };
+    let csrf_cookie = getCookie("CSRF-TOKEN");
+    if (csrf_cookie) {
+      headers["X-CSRF-Token"] = csrf_cookie;
+    }
+    fetch(`${members_path}/${memberId2.value}`, {method: "DELETE", headers: headers}).then((response) => {
+      if (response.status === 200) {
+        response.json().then((data) => {
+          resultsDiv.innerHTML = "";
+          let parag = document.createElement("P");
+          parag.textContent = JSON.stringify(data);
+          resultsDiv.appendChild(parag);
+        });
+      } else {
+        response.json().then((data) => {
+          resultsDiv.innerHTML = "";
+          let parag = document.createElement("P");
+          parag.textContent = JSON.stringify(data);
+          resultsDiv.appendChild(parag);
+        });
+      }
+    })
+
```

Comment:
This looks great! Great work keeping things well organized and in a similar pattern to the code that was already present in the file. The only change I might recommend her would be to create some line breaks on line 162 to make it a little bit more readable. I.e.
```
fetch(
   `${members_path}/${memberId2.value}`, 
      {method: "DELETE", headers: headers}
).then...
```